M2/Macaulay2/m2/webapp.m2

```diff
@@ -116,3 +129,53 @@ CoherentSheaf#{WebApp,AfterPrint} = F -> (
  )
 
 ZZ#{WebApp,AfterPrint} = identity
+
+if topLevelMode === WebApp then (
+    (webAppEndTag,            -- closing tag
+	webAppHtmlTag,        -- indicates what follows is HTML
+	webAppOutputTag,      -- it's html but it's output
```

Comment:
Why not always set these to these values, in advance?  And not just when topLevelMode is WebApp?

Reply:
yeah that's the most debatable change I introduced. Let me think about it again. The idea is that internally the actual web app uses those shortened, somewhat obscure tags (ascii characters < 32) which are then turned into normal html tags. Turning on topLevelMode manually would lead to a sort of "debug mode" where the tags appear as (a rough approximation of) their final appearance.
But the truth is, I don't see any real application for this. Let me think about it some more and possibly revert it.

Reply:
reverted